src/test/java/example/TraverseDemoTest.java

```diff
@@ -0,0 +1,57 @@
+package example;
+
+import org.junit.jupiter.api.*;
+import org.neo4j.driver.*;
+import org.neo4j.harness.Neo4j;
+import org.neo4j.harness.Neo4jBuilders;
+
+import java.io.File;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class TraverseDemoTest {
+
+    private static final Config driverConfig = Config.builder().withoutEncryption().build();
+    private Neo4j embeddedDatabaseServer;
+
+    @BeforeAll
+    void initializeNeo4j() {
+        this.embeddedDatabaseServer = Neo4jBuilders.newInProcessBuilder()
+                .withProcedure(TraverseDemo.class)
+                .withFixture(new File(getClass().getResource("/movie.cypher").getPath()))
+                .build();
+    }
+
+    @Test
+    void findKeanuReevesCoActors() {
+
+        try(
+                Driver driver = GraphDatabase.driver(embeddedDatabaseServer.boltURI(), driverConfig);
+                Session session = driver.session()) {
+
+            List<String> names = session.run("match (keanu:Person {name:'Keanu Reeves'})-[*1..2]-(coactors:Person)\n" +
```

Comment:
We can also use `var` types. 

```suggestion
            var names = session.run("match (keanu:Person {name:'Keanu Reeves'})-[*1..2]-(coactors:Person)\n" +
```

Reply:
Thanks. Guess I'm still stuck in the 'old' world. Not doing enough development right now :-(